demo/index.html

```diff
@@ -197,6 +197,25 @@
         </table>
       </div>
       <div>
+        <h2>Add a button to your toast</h2>
+        <table>
+          <thead>
+            <tr>
+              <td>Attribute</td>
+              <td>Value</td>
+              <td>Result</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>buttonText</td>
+              <td>Button</td>
```

Comment:
Here, the value is "Button" and in the fired event the actual value is "Press Me".